kura/org.eclipse.kura.linux.watchdog/src/main/java/org/eclipse/kura/linux/watchdog/WatchdogServiceImpl.java

```diff
@@ -72,7 +74,7 @@ public void updated(Map<String, Object> properties) {
         if (this.options.isEnabled()) {
             this.rebootCauseWriter = new RebootCauseFileWriter(this.options.getRebootCauseFilePath());
 
-            try (PrintWriter wdWriter = new PrintWriter(this.options.getWatchdogDevice());) {
+            try (PrintWriter wdWriter = new PrintWriter(WD_FILE);) {
```

Comment:
The `watchdogDevice` configuration property and the `WD_FILE` constant refer to two different files:
* the first property contains the the path of the watchdog device to be used e.g. `/dev/watchdog`.
* The second constant defines the path of a file that is created if the watchdog is enabled and contains the path of the watchdog device, this file is used for example by the init scripts and the upgrader.

In my previous pull request I've refactored the configuration options handling by creating the `WatchdogServiceOptions` class and I've accidentally removed the references to WD_FILE, this PR fixes the issue.
Maybe we could define a more descriptive name for the `WD_FILE` constant in order to avoid confusing it with the `watchdogDevice` property.